sigs.yaml

```diff
@@ -1611,13 +1616,19 @@ sigs:
       - name: sig-scalability-test-failures
         description: Test Failures and Triage
     subprojects:
-    - name: kubemark
+    - name: kubernetes-scalability-definition
       owners:
-      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/cmd/kubemark/OWNERS
-      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubemark/OWNERS
-    - name: perf-tests
+      - https://github.com/kubernetes/community/blob/master/sig-scalability/slos/OWNERS
+    - name: scalability-governance
+      owners:
+      - https://github.com/kubernetes/community/blob/master/sig-scalability/governance/OWNERS
+    - name: scalability-governance
       owners:
-      - https://raw.githubusercontent.com/kubernetes/perf-tests/master/OWNERS
+      - https://github.com/kubernetes/community/blob/master/sig-scalability/governance/OWNERS
+    - name: scalability-test-frameworks
+      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubemark/OWNERS
+      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/test/kubemark/OWNERS
+      - https://raw.githubusercontent.com/kubernetes/perf-tests/blob/master/clusterloader2/OWNERS
```

Comment:
Aren't we the owners for perf-tests root? Maybe except something around networking (which we may also assume)?

Reply:
We are. I wanted to mention clusterloader2 explicitly. But I'm happy to add perf-tests/owners too here.